plugins/github/server/processors/IntegrationDeletedProcessor.ts

```diff
@@ -0,0 +1,62 @@
+import { App, Octokit } from "octokit";
+import { IntegrationService, IntegrationType } from "@shared/types";
+import { Integration } from "@server/models";
+import BaseProcessor from "@server/queues/processors/BaseProcessor";
+import { IntegrationEvent, Event } from "@server/types";
+import { GitHub } from "../github";
+
+export default class IntegrationDeletedProcessor extends BaseProcessor {
+  static applicableEvents: Event["name"][] = ["integrations.delete"];
+
+  private githubClient?: Octokit;
+
+  constructor() {
+    super();
+    if (!GitHub.appId || !GitHub.appPrivateKey) {
```

Comment:
This is going to stop integrations from deleting properly if GitHub isn't setup, seems like a mistake?

Reply:
Tried Slack after disabling GitHub. Deletion seems to be working fine. Sorry, how would it fail?